src/utils/method-helpers.js

```diff
@@ -28,3 +29,6 @@ const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
 
 export const delay = (delayed, ms) =>
   Promise.all([delayed, sleep(ms)]).then(([data]) => data);
+
+export const translate = (key, lang, interpolation = null) =>
```

Comment:
Ah I see what you mean. I agree it would definitely be nicer not having to pass it down as a parameter _every time_. The only thing is I have it as part of the redux store just like every other attribute in the app so it makes it easy to see how a language is changed and so forth. As you mentioned, I think we could manually set it to `AsyncStorage` on language change and then manually retrieve that parameter from there in the translate method but that sort of abstracts things away from the redux store (and I kinda like keeping everything attached to the store if possible).

Will you be okay with having the way it is now and opening a separate refactor ticket to discuss? Definitely think I may be overthinking things and your approach or something similar but be nicer but I'm not sure.